CHANGELOG.md

```diff
@@ -7,11 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Added ELB Logging to Stack Builder with bucket and policy creation.
```

Comment:
You need to add the change type header. Have a look at the other section, they have the change type headers.

Reply:
Added the ### Added line if that's what you meant